e2e_tests/fixtures/util.js

```diff
@@ -0,0 +1,24 @@
+const shell = require('shelljs');
+const path = require('path');
+const rootDir = process.cwd();
+
+shell.config.silent = true;
+
+module.exports = {
+  setupStageWithFixture: (stageName, fixtureName) => {
+    const stagePath = path.join(rootDir, stageName);
+    shell.mkdir(stagePath);
+    shell.exec(`cp -a ${rootDir}/e2e_tests/fixtures/${fixtureName}/. ${stagePath}/`);
```

Comment:
I tried super hard to get it to work with `shell.cp` but it wouldn't actually copy the files `/.` into the directory. :man_shrugging: